apps/web/components/eventtype/AssignmentWarningDialog.tsx

```diff
@@ -0,0 +1,44 @@
+import { useRouter } from "next/navigation";
+import type { Dispatch, SetStateAction } from "react";
+import type { MutableRefObject } from "react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { Dialog, ConfirmationDialogContent } from "@calcom/ui";
+
+interface AssignmentWarningDialogProps {
+  isOpenAssignmentWarnDialog: boolean;
+  setIsOpenAssignmentWarnDialog: Dispatch<SetStateAction<boolean>>;
+  pendingRoute: string;
+  leaveWithoutAssigningHosts: MutableRefObject<boolean>;
+}
+const AssignmentWarningDialog = (props: AssignmentWarningDialogProps) => {
+  const { t } = useLocale();
+  const {
+    isOpenAssignmentWarnDialog,
+    setIsOpenAssignmentWarnDialog,
+    pendingRoute,
+    leaveWithoutAssigningHosts,
+  } = props;
+  const router = useRouter();
+  return (
+    <Dialog open={isOpenAssignmentWarnDialog} onOpenChange={setIsOpenAssignmentWarnDialog}>
```

Comment:
You know what would be cool, if `Go Back and Assign` is clicked, that it re-directs to the `/event-types/:id?tabName=team` to land user on the page where he / she can assign the user. Otherwise the user needs to figure out which tab to press for assignement.

<img width="611" alt="Screenshot 2024-05-17 at 10 15 04" src="https://github.com/calcom/cal.com/assets/42170848/535cafd1-a9b0-4e0c-801f-49c84affb77b">

Reply:
sure Lauris, that's a cool idea.

Reply:
Amazing! Thank you Anik!